src/shared/components/app/app.tsx

```diff
@@ -26,7 +26,7 @@ export class App extends Component<AppProps, any> {
       <>
         <Provider i18next={i18n}>
           <div>
-            <Theme myUserInfo={siteRes.my_user} />
+            <Theme myUserInfo={siteRes.my_user} site={siteRes.site_view.site} />
```

Comment:
I'd prefer you just pass in the default theme from site, and have the param be something like `defaultSiteTheme` .

My convention, and I'm doing this for android too, is to always use the lowest level possible, then once you need extra data from a few levels up, then select the bigger dataset. But only do that if you really need that other data.